.circleci/config.yml

```diff
@@ -26,12 +42,14 @@ aliases:
   - &env_postgresql
     environment:
       MIGRATION_DIR: migrations/postgres
+      MIGRATION_DIRECTORY: migrations/postgres
```

Comment:
Isn't there one of the two lines above that should be removed?

Reply:
gosh my config is so old I had MIGRATION_DIR in it (so I copied it), but I was missing the MIGRATION_DIRECTORY for diesel so I added it, but in fact MIGRATION_DIR is no longer used by us